actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/UserBulkUploadBackgroundJobActor.java

```diff
@@ -260,36 +257,36 @@ private void callCreateUser(User user, BulkUploadProcessTask task, String orgNam
           task,
           ProjectUtil.BulkProcessStatus.FAILED,
           ResponseCode.internalError.getErrorMessage(),
-          row,
+              user,
           JsonKey.CREATE);
     } else {
-      row.put(JsonKey.ID, userId);
-      row.put(JsonKey.ORG_NAME, orgName);
-      setSuccessTaskStatus(task, ProjectUtil.BulkProcessStatus.COMPLETED, row, JsonKey.CREATE);
+      user.put(JsonKey.ID, userId);
+      user.put(JsonKey.ORG_NAME, orgName);
+      setSuccessTaskStatus(task, ProjectUtil.BulkProcessStatus.COMPLETED, user, JsonKey.CREATE);
     }
   }
 
   @SuppressWarnings("unchecked")
-  private void callUpdateUser(User user, BulkUploadProcessTask task, String orgName)
+  private void callUpdateUser(Map<String, Object> user, BulkUploadProcessTask task, String orgName)
       throws JsonProcessingException {
     ProjectLogger.log("UserBulkUploadBackgroundJobActor: callUpdateUser called", LoggerEnum.INFO);
-    Map<String, Object> row = mapper.convertValue(user, Map.class);
+    //Map<String, Object> row = mapper.convertValue(user, Map.class);
```

Comment:
remove commented code//
and use eithe user object or map not both.

Reply:
Removed.
I followed with map here, reason is from request we are getting map and again we sending as a part of request. Both will have map as attribute so considered map.